assets/js/components/notifications/BannerNotifications.js

```diff
@@ -53,15 +56,27 @@ export default function BannerNotifications() {
 
 	return (
 		<Fragment>
-			{ ( 'authentication_success' === notification ||
-				'user_input_success' === notification ) && (
-				<SetupSuccessBannerNotification />
+			{ ! viewOnly && (
+				<Fragment>
+					{ ( 'authentication_success' === notification ||
+						'user_input_success' === notification ) && (
+						<SetupSuccessBannerNotification />
+					) }
+					{ isAuthenticated && <CoreSiteBannerNotifications /> }
```

Comment:
This should be redundant as `! viewOnly` implies authentication. This shouldn't break anything but we probably need to establish some criteria for when to use view context rather than authentication status.